docs/layers/grid-layer.md

```diff
@@ -5,19 +5,21 @@
   <img src="https://img.shields.io/badge/extruded-yes-blue.svg?style=flat-square" alt="extruded" />
 </p>
 
-# GridLayer
+# Grid Layer
```

Comment:
Ok let's go with the actual layer names then (no spaces). It will match better with the API reference docs, the examples etc.